### PR TITLE
Create carto package json

### PR DESCRIPTION
**Pull request by javitonino**
https://github.com/CartoDB/onpremises/issues/560

:eyes: @lekum @ivanmalagon 

```
{
  "builder": {
    "repo_url": "https://github.com/CartoDB/cartodb.git",
    "tag_format": "v{VERSION}",
    "manifest": "carto-package.json"
  }
}
```

### CODE REVIEW

**Comment by lekum** [comment]
Would it be possible to use semver with the three period-separated number instead of abbrevitated versions as 3.0?

**Comment by javitonino** [comment]
Not all packages have patch versions. e.g: zip, unp, postgresql (version 10+) only use two numbers for versioning. I mean, I can add a `.0` but it won't always make sense.

File: package.json [hunk 1]
```diff
@@ -179,10 +179,10 @@
...
     "lint:css": "stylelint './app/assets/stylesheets/**/*.scss'",
     "lint:fix": "eslint . --fix",
-    "bump": "npm version patch",
-    "bump:major": "npm version major",
-    "bump:minor": "npm version minor",
-    "postversion": "git push origin master --follow-tags",
+    "bump": "npm --no-git-tag-version version patch",
+    "bump:major": "npm --no-git-tag-version version major",
+    "bump:minor": "npm --no-git-tag-version version minor",
+    "postversion": "git add package.json && git commit -m 'Bump assets version' && git push origin master",
```
Comment (javitonino) [inline package.json]: @ivanmalagon Please take a look

**Comment by ivanmalagon** [comment]
Jumping on board to split the assets and Builder versions.

**Pull request merged**